modules/mri_violations/php/NDB_Menu_Filter_Form_resolved_violations.class.inc

```diff
@@ -0,0 +1,453 @@
+<?php
+/**
+ * This page implements a front end which is used to unify all
+ * of the different MRI violation types (could not identify scan,
+ * header violation in the scan after identifying scan type, mislabeled 
+ * scan, etc) and link to the appropriate module to further investigate
+ * why a scan was excluded by the imaging pipeline scripts.
+ *
+ * PHP Version 5
+ *
+ * @category MRI
+ * @package  Main
+ * @author   Dave MacFarlane <driusan@bic.mni.mcgill.ca>
+ * @license  GPLv3
+ * @link     https://www.github.com/aces/Loris-Trunk/
```

Comment:
This header is the same as the previous page, one of them is wrong (I think this one, since I didn't write it..)